src/core/qgshtmlutils.cpp

```diff
@@ -0,0 +1,17 @@
+#include "qgshtmlutils.h"
+
+QString QgsHtmlUtils::buildBulletList( const QStringList &values )
+{
+  QString s( QStringLiteral( "<ul>" ) );
+
+  for ( QStringList::const_iterator i = values.begin();
+        i != values.end();
+        ++i )
+  {
+    s += QLatin1String( "<li>" ) + *i + QLatin1String( "</li>" );
```

Comment:
If in doubt, use a template based approach `QStringLiteral( "<li>%1</li>" ).arg( value );`